src/api/java/com/minecolonies/api/util/ItemStackUtils.java

```diff
@@ -362,6 +363,31 @@ public static int getItemStackSize(final ItemStack stack)
     }
 
     /**
+     * set the size of the stack.
+     * This is for compatibility between 1.10 and 1.11
+     *
+     * @param stack to set the size to
+     * @param size of the stack
+     */
+    @NotNull
+    public static void setSize(@NotNull final ItemStack stack, final int size)
+    {
+        stack.setCount(size);
+    }
+
+    /**
+     * Increase or decrease the stack size.
+     *
+     * @param stack to set the size to
+     * @param amount to increase the stack's size of (negative value to decrease)
+     */
+    public static void increaseOrDecreaseSize(@NotNull final ItemStack stack, final int amount)
```

Comment:
I think changeSize is okay.
Even if I personally prefer a separate increase and decrease method.